docker/Dockerfile

```
@@ -0,0 +1,107 @@
+FROM --platform=linux/amd64 node:16.13.0 AS frontend
+
+WORKDIR /app/oppia
+
+RUN apt-get update -y && apt-get upgrade -y \
+    curl \
+    git \
+    npm
```

```suggestion
RUN apt-get update -y && apt-get upgrade -y curl git npm
```

Also, do we need to install npm?

---

yeah, npm is install yarn for the installing the FE packages

---

I was assuming node image to come with npm

---

yes, we can remove npm, I checked that. thanks